components/butterfly/src/member.rs

```diff
@@ -62,7 +62,8 @@ impl FromStr for Health {
             "alive" => Ok(Health::Alive),
             "suspect" => Ok(Health::Suspect),
             "confirmed" => Ok(Health::Confirmed),
-            _ => Ok(Health::Alive),
+            "departed" => Ok(Health::Departed),
+            value => panic!("No match for Health from string, {}", value),
```

Comment:
Exactly! We shouldn't be scared to assert in the Supervisor since we're testing things regularly in acceptance